<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_source_selection</class>
 <widget class="QWidget" name="widget_source_selection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select stream</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonWebcam">
       <property name="text">
        <string>Webcam</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxWebCam"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButtonVideo">
       <property name="text">
        <string>Video</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditVideo"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonOpenVideoFolder">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButtonRTSP">
       <property name="text">
        <string>rtsp</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditRTSP">
       <property name="placeholderText">
        <string>rtsp://192.168.1.2:8080/out.h264</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButtonWebsockets">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>websockets</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditWebsockets">
       <property name="text">
        <string>ws://127.0.0.1:1234</string>
       </property>
       <property name="placeholderText">
        <string>ws://127.0.0.1:1234</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButtonHLS">
       <property name="text">
        <string>HLS/DASH</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditHLS">
       <property name="placeholderText">
        <string>http://techslides.com/demos/sample-videos/small.mp4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelMaxFps">
       <property name="text">
        <string>Max fps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderMaxFPS">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMaxFPS">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>420</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMaxFPS</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxMaxFPS</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMaxFPS</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderMaxFPS</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
